src/eko/anomalous_dimensions/as4/ggg.py

```diff
@@ -21,7 +23,7 @@ def gamma_gg_nf3(n, sx):
     Returns
     -------
     complex
-            |N3LO| non-singlet anomalous dimension :math:`\\gamma_{gg}^{(3)}|_{nf^3}`
+        |N3LO| non-singlet anomalous dimension :math:`\\gamma_{gg}^{(3)}|_{nf^3}`
```

Comment:
Just to make it easier (and slightly more consistent), you can use a single `\` if you prepend an `r` to the string:
```py
r""":math:`\alpha`"""
```
It is a raw string, and the escaped character are not interpreted (so if you write a `\n` it will correspond to backslash and the letter, instead of a newline).